account-kit/signer/src/base.ts

```diff
@@ -740,30 +780,49 @@ export abstract class BaseAlchemySigner<TClient extends BaseSignerClient>
       const existingUser = await this.getUser(params.email);
       const expirationSeconds = this.getExpirationSeconds();
 
-      const { orgId, otpId } = existingUser
-        ? await this.inner.initEmailAuth({
-            email: params.email,
-            emailMode: params.emailMode,
-            expirationSeconds,
-            redirectParams: params.redirectParams,
-          })
-        : await this.inner.createAccount({
-            type: "email",
-            email: params.email,
-            emailMode: params.emailMode,
-            expirationSeconds,
-            redirectParams: params.redirectParams,
-          });
+      if (existingUser) {
```

Comment:
This whole method should probably be broken up into multiple methods but I figured I could do that as a separate task if we want to circle back to it